.github/workflows/docs.yml

```diff
@@ -33,17 +36,31 @@ jobs:
           virtualenvs-create: true
           virtualenvs-in-project: true
 
-      - name: Cache datasets and docs
-        id: cached-dataset-docs
+      - name: Create/Restore MNE Data Cache
+        id: cache-mne_data
         uses: actions/cache@v3
         with:
-          key: doc-${{ github.head_ref }}-${{ hashFiles('moabb/datasets/**') }}
-          path: |
-            ~/mne_data
-            docs/build
+          path: ~/mne_data
+          key: ${{ runner.os }}-mne_data
+
+      - name: Cache docs build
+        id: cache-docs
+        uses: actions/cache@v3
+        with:
+          key: docs-build-${{ github.run_id }}-${{ github.run_attempt }}
```

Comment:
```suggestion
          key: docs-build
```

Reply:
I used `docs-build-${{ github.run_id }}-${{ github.run_attempt }}` to have one different cache for every new CI run. Otherwise, some previously existing builds will be re-used. Is this what we want?

Reply:
Usually yes, in almost all cases we want to reuse installed pip/poetry packages and downloaded datasets